backend/booking_engine/README.md

```diff
@@ -1,49 +1,139 @@
-![Chronogolf][crest]
+![Lightspeed Golf][crest]
 
 # The Booking Engine challenge
 
-Golf clubs manage player reservations through our system. When a player wants to
+## Intro to Building a Booking Engline
+You can use this prompt to introduce the the challenge and explain a little about our app:
+> Golf clubs manage player reservations through our system. When a player wants to
 make a reservation, they request it through a booking engine. The system will
 then confirm if the submitted reservation is valid.
-
-Your mission today consists of building a simple booking application that
+>
+> Your mission today consists of building a simple booking application that
 validates reservation requests and returns a list of confirmed reservations.
 
-### Environment Setup
+</br>
 
-Let's take care of your setup first.
+As you interview, be mindful of the following things overall:
 
-- Choose a dev environment and a language that suits you
-  - A dev environment on your personal computer is perfect!
-  - Coderpad is available if you would prefer
-  - Choose the language you are most comfortable with
-- Let's start pairing together!
+- Is the candidate a person we would enjoy working with?
+- Is the candidate receptive to feedback?
+- Did the candidate showcase any toxic or inappropriate behaviour?
+- Will the candidate be interested in the work and culture we can provide?
```

Comment:
I gotta say, I kinda feel like this sort of thing doesn't belong in the README.

In my experience doing these technical interviews, I walk the candidate through this README to get things set up and understand the challenge together. While the candidate is looking through this, I don't feel like they should see what we're looking for on our side, and I don't think it really helps the candidate either (could contribute to nervousness or make them overly conscious of how they're adhering to these things). 

Also worth mentioning that since this a public repo, the candidate could find this before the interview and then come into it knowing what we're looking for. Perhaps we don't see that as a problem, just pointing it out.

I believe there's something similar on the Greenhouse side already (I could be wrong). If not, I think that might be where it belongs, so perhaps there's somewhere there where we can add it, to remind ourselves what we're looking for out of the candidate.

Reply:
oh interesting, i didn't know we shared this repo to the candidate though that makes sense! but if thats the case, i don't think they should see this info at all either. more so these additions were intended to shift the readme to be a developer guide.

i can see a use case for having a clean file that just outlines the challenge to the candidate AND a file that is to help guide the developers.

Reply:
Yeah I like that idea. It's also possible I use the repo differently than others, but I usually hand the candidate a link to this README so we can go through it together, so it jumped out to me immediately. In that case I'd definitely prefer a separate file we can use as a reference we can use while interviewing the candidate.

Reply:
Same here actually, I used to deliver the Readme to the candidate but I was wondering how bad it is for them to see this, or if we ever want to share the file with them rather than the steps.
I also like the idea of separating them, will go with that so the interviewers can follow the flow they want. Good call!